java/code/src/com/redhat/rhn/frontend/xmlrpc/LoggingInvocationProcessor.java

```diff
@@ -33,8 +35,9 @@
  * LoggingInvocationProcessor extends the marquee-xmlrpc library to allow
  * us to log method calls.
  */
-public class LoggingInvocationProcessor implements XmlRpcInvocationInterceptor {
+public class LoggingInvocationProcessor extends BaseLoggingInvocationProcessor implements XmlRpcInvocationInterceptor {
     private static Logger log = LogManager.getLogger(LoggingInvocationProcessor.class);
+
     private static ThreadLocal<User> caller = new ThreadLocal<>();
 
     private static ThreadLocal timer = new ThreadLocal() {
```

Comment:
I think `timer` can also be moved up to the base class. From there you can either expose the whole object via a `protected StopWatch getTimer()` method or maybe prefer to expose individual methods for each required operation (start, stop, reset, etc.)